Program.cs

```diff
@@ -356,7 +381,7 @@ private void fightDescriptionWin( Monster monster )
         {
             if (knightOrMage == "m")
             {
-                var lines = LoadCSVFile("./Config/WinDescriptions-Knight.csv");
+                var lines = LoadCSVFile("./Config/WinDescriptions-Mage.csv");
```

Comment:
nicely spotted - I must have missed this in the last review ;)

Reply:
I found that the hard way